### ISSUE TITLE
Uncaught (in promise) TypeError: 'get' on proxy: property 'labels' is a read-only and non-configurable data property on the proxy target but the proxy did not return its actual value (expected '[object Array]' but got '[object Array]')

### ISSUE THREAD

**Issue by Credit977** [bpmn-io/diagram-js#925]
### Describe the Bug

在使用 bpmn-js中的Modele里的 updateProperties方法来修改xml中的节点信息时

控制台中报错了diagram-js/lib/model/index.js  里的  
 Object.defineProperty(this, 'label', {
    get:  function() { 
      return this.labels[0];
      },
    set: function(newLabel) {

      var label = this.label,
          labels = this.labels;

      if (!newLabel && label) {
        labels.remove(label);
      } else {
        labels.add(newLabel, 0);
      }
    }
  });
会报错  Uncaught (in promise) TypeError: 'get' on proxy: property 'labels' is a read-only and non-configurable data property on the proxy target but the proxy did not return its actual value (expected '[object Array]' but got '[object Array]')
请问这是因为节点信息是空才会这样报错吗？

### Steps to Reproduce

1. do this
2. do that

If possible, try to build [a test case](https://github.com/bpmn-io/diagram-js/tree/main/test/spec) that reproduces your problem.

### Environment

Please complete the following information:

 - Browser: [e.g. IE 11, Chrome 69]
 - OS: [e.g. Windows 7]
 - Library version: [e.g. 2.0.0]

**Comment by nikku** [comment]
@Credit977 Unfortunately your report is not actionable for us. Please rephrase it to provide clear steps to reproduce or we cannot help you.

**Comment by qingtian5266** [comment]
一样的问题，咋解决啊
